examples/encrypting.rs

```diff
@@ -52,11 +52,10 @@ async fn initialize_sdk_from_file(device_path: &PathBuf) -> Result<(DeviceContex
         let ironoxide = ironoxide::initialize(&device_context, &Default::default()).await?;
         Ok((device_context, ironoxide))
     } else {
-        panic!(format!(
+        panic!(
```

Comment:
This was a change Colt did on the website's example file but forgot to do the ironoxide's file